### ISSUE TITLE
[Link and button labels] Button and link language isn't descriptive and/or their purpose isn't clear. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29356]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Peggy Gannon (Content)

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
"Highlight differences" label should be more descriptive and give user some basic context as to what happens if users check that box. 

### VA.gov Experience Standard
[Cateogy Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

---

### Platform Recommendation
Provide some content about what users will see when they check that box. 

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by amyknox** [comment]
@peggygannon - Our research into comparison UI's showed "Highlight differences" was a standard CTA for adding emphasis to fields that vary between selected appliances, auto features, cosmetic products, etc. So we'd like to keep the "Highlight differences" label as-is.  

Mikki had a suggestion on how to make the interaction even more obvious.  Slightly paraphrasing it here, "... show the asterisk with text next to the content:  "An * indicates the information is different between your selected institutions."

Curious to get your thoughts on that.

**Comment by amyknox** [comment]
@peggygannon 

**Updating:**
* Heading = Compare institutions (breadcrumb will need to be updated as well)

**Recommending:**
* Using the recommendation for "*Highlight differences" checkbox subtext as an aria label on the checkbox but not including it in the visual.  Our testing indicates that users do not have any trouble understanding the difference between institution categories when the asterisk + color are used. We're also concerned that adding the extra text will crowd the control and causing users to overlook it. 

**Not updating:** 
"Compare benefits at approved schools, employers and VET TEC providers." since the comparison is greater than just benefits (location, accreditation, school size)

**Comment by watersemma** [comment]
Updates have been made and can be seen here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/school-comparison-tool/redesign/design/sketch-files/gibct-redesign-comparison-postmvp.sketch

**Comment by af-manish** [comment]
@peggygannon <cc @ERRonat0609 @mrblanco1 > Gentle nudge here.

**Comment by af-manish** [comment]
@ERRonat0609  Any thoughts on this? Can we get a pdf?

**Comment by ERRonat0609** [comment]
This is a pdf of what is in the above-linked sketch file.
[gibct-redesign-comparison-postmvp.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/7549788/gibct-redesign-comparison-postmvp.pdf)

**Comment by mrblanco1** [comment]
Gina to follow up with Peggy.

**Comment by peggygannon** [comment]
@ERRonat0609  and @mrblanco1 Hello! I am looking at the PDF, but I don't see this update from Amy Knox's 9/15 comment  
1. **Updating**

Heading = Compare institutions (breadcrumb will need to be updated as well)

2. In addition, on screens 29 and 30 in the PDF, I'm seeing two different heads: 
screen 29 says: Institution comparison
screen 30 says: School comparison

Which screen is correct?

**Comment by RLHecht** [comment]
@af-manish Could you update the PDF based on Peggy's last comment? Also, the headers need to be sentence capped so "comparison" should be lowercase (page 39 and the bottom of page 7 as examples).

**Comment by mrblanco1** [comment]
Need to reach out to Platform to request a new sketch file. @af-manish to follow up with Sean.

**Comment by af-manish** [comment]
@seanasewell  Sean, can you help update the Sketch file here so we can get the PDF?

**Comment by shiragoodman** [comment]
hey @af-manish  - Sean is out today so I'm stepping in. The Platform is not responsible for updating the file in Sketch. If you need access to Sketch, please check out [Sketch for Teams at VA](https://design.va.gov/documentation/designers#sketch-for-teams-at-va).

**Comment by af-manish** [comment]
@RLHecht  Attached is the updated PDF and Sketch file with yours and Peggy's comments.
[gibct-redesign-comparison-postmvp-12_20_21 updates.sketch.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/7756437/gibct-redesign-comparison-postmvp-12_20_21.updates.sketch.zip)

[gibct-redesign-comparison-postmvp-12_20_21 updates.pdf.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/7756439/gibct-redesign-comparison-postmvp-12_20_21.updates.pdf.zip)

**Comment by RLHecht** [comment]
@af-manish In the PDF, pages 34 and 35 are still showing "School comparison" as the header instead of "Institution comparison." And in the mobile views on pages 36, 37, 39, 40, and 41, the header isn't sentence cased. It should be "Institution comparison" (comparison is lowercase)

**Comment by af-manish** [comment]
@RLHecht  Thx. Attached are updated files. Please check.
[gibct-redesign-comparison-postmvp-12_20_21 updates (1).sketch.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/7769993/gibct-redesign-comparison-postmvp-12_20_21.updates.1.sketch.zip)
[gibct-redesign-comparison-postmvp-12_20_21 updates (1).pdf.zip](https://github.com/department-of-veterans-affairs/va.gov-team/files/7769996/gibct-redesign-comparison-postmvp-12_20_21.updates.1.pdf.zip)

**Comment by RLHecht** [comment]
@af-manish These changes in the PDF look good. Thanks!

**Comment by af-manish** [comment]
Thx @RLHecht

**Comment by hallihocine** [comment]
Ticket has been validated in staging: the header on the compare page is now updated to "Institution comparison" and the breadcrumb accurately reflects the new head as well. The intro text under the header also updated to: View school information side by side to compare schools.
<img width="1012" alt="Screen Shot 2022-02-15 at 10 52 16 AM" src="https://user-images.githubusercontent.com/86266024/154111491-b1892f5b-0240-4442-a333-a8be1499cea1.png">

**Comment by hallihocine** [comment]
- Test run in Test rail:
https://dsvavsp.testrail.io/index.php?/runs/view/2902&group_by=cases:section_id&group_order=asc&group_id=3649
- Test Steps: 
[Button and link language isn't descriptive and their purpose isn't clear.xlsx](https://github.com/department-of-veterans-affairs/va.gov-team/files/8073576/Button.and.link.language.isn.t.descriptive.and.their.purpose.isn.t.clear.xlsx)

**Comment by mrblanco1** [comment]
QA & UAT Passed on 2/18/2022. Closing.
